.github/workflows/release.yml

```
@@ -8,16 +8,47 @@ on:
+        run: |
+          gh run cancel ${{ github.run_id }}
+          gh run watch ${{ github.run_id }}
       - name: Checkout
         uses: actions/checkout@v4
       - name: Setup Rust
         uses: dtolnay/rust-toolchain@stable
         with:
           targets: 'armv7-linux-androideabi, i686-linux-android, aarch64-linux-android, x86_64-linux-android'
+      - name: Retrieve cache
+        uses: Leafwing-Studios/cargo-cache@v2
+        with:
```

actionlint (1.7.4)

"with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)